store-tests

```diff
@@ -72,7 +72,7 @@ def main():
     statuses = api.all_statuses(opts.revision)
     if not statuses:
         logging.warning("Revision %s has no statuses", opts.revision)
-        return 0
+        return
```

Comment:
This case is probably better written as `return f"Revision {opts.revision} has no statuses"`.  In case `sys.exit()` gets a string, it'll write it to stderr and exit non-zero.

Reply:
But then we again have the case that some exit paths return a value and some don't. Also, this is a potential breakage in our bots (if the command fails). I can either take out the commit, and block #4039 further, or we do this in a follow-up?

Reply:
Conceptually is this an error, or a valid case where we merely want to issue a warning?

If it's the latter, then probably we should leave the code as it is..

In any case, if our `main()` function is always returning `None` then it makes absolutely no sense to pass its return value to `sys.exit()`.  So drop the `sys.exit()` wrapping of `main()` at the bottom of the file.  And if you want to deal with this case as an error then you can just use `sys.exit()` directly here...

Reply:
I can't say off-hand whether this should be fatal and crash the bot. My gut feeling is "fatal", but that'd be a behaviour change. But I'm willing to try. Reworked.